src/structures/emoji/GuildPreviewEmoji.ts

```diff
@@ -0,0 +1,8 @@
+import { Snowflake } from 'discord-api-types';
+import { BaseGuildEmoji } from './BaseGuildEmoji';
+
+export class GuildPreviewEmoji extends BaseGuildEmoji {
+	public get roles(): Set<Snowflake> {
```

Comment:
why not directly on a BaseGuildEmoji ?

Reply:
Because GuildEmoji should have the same method, but instead of returning a Set<Snowflake>, it should return a Collection<Snowflake, Role>

Reply:
okay.